src/main/java/com/desiato/puresynth/configurations/DataInitializer.java

```diff
@@ -0,0 +1,28 @@
+package com.desiato.puresynth.configurations;
+
+import com.desiato.puresynth.models.User;
+import com.desiato.puresynth.repositories.UserRepository;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
```

Comment:
What's. the purpose of that? After you create one user, it should be saved in the DB. Why do you want to initialize it in this class?